Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -76,8 +76,17 @@ protected async Task HandleReadData( CancellationToken cancellationToken = defau
             {
                 Loading = true;
 
+                // should remove IsTextSearchable and let ReadData decide whetehr text is searchable or not?
```

Comment:
I don't know. Mixed feelings about this one.
But I think it should be kept. It basically just does this. WHy would you not call ReadData only when the `MinLen` requirement has been achieved? 
![image](https://user-images.githubusercontent.com/22283161/182021633-98243031-cd0b-4e00-9c29-0194f0b78858.png)

@stsrki  toughts?

Removed the comment after decision.

Reply:
Yeah, I'm also not sure we should clear the filtered data here.